app/src/main/java/com/example/houta/dadvice_v01/MainActivity.java

```diff
@@ -139,9 +139,9 @@ private void setUpAndLoadNativeExpressAds() {
             public void run() {
                 final float density = MainActivity.this.getResources().getDisplayMetrics().density;
                 // Set the ad size and ad unit ID for each Native Express ad in the items list.
-                for (int i = 0; i <= mRecyclerViewItems.size(); i += ITEMS_PER_AD) {
+                for (int i = ITEMS_PER_AD; i <= mRecyclerViewItems.size(); i += ITEMS_PER_AD) {
```

Comment:
Making i = 0 into i = ITEMS_PER_AD makes it so that instead of the ad showing up at index 0 (top), it shows up at least one set down (ex 8 spots down). I like it.